ftw/poodle/browser/poodletable.py

```diff
@@ -43,6 +44,15 @@ def isCurrentUser(self, userid):
         # also add is poodle active check
         return user.id == userid and self.is_active()
 
+    def isOwner(self):
```

Comment:
This function does not what I do expect.

For me, it should check if I am the owner of the current context or not. But actually, it checks the stealth_voting-attribute and the owner.

Please rename the function to `show_results` or something like that.

Also the comment is not really true. If the stealth_voting attribute is disabled, the function will return True without checking the user anymore.